src/navigation.ts

```diff
@@ -432,6 +440,32 @@ export class Navigation {
     }
   }
 
+  /**
+   * Handle the widget or dropdown div losing focus (via focusout).
+   *
+   * Because we skip the widget/dropdown div cases in `handleBlurWorkspace` we need
+   * to catch them here.
+   *
+   * @param workspace The workspace.
+   * @param relatedTarget The related target (newly focused element if any).
+   */
+  handleFocusOutWidgetDropdownDiv(
+    workspace: Blockly.WorkspaceSvg,
+    relatedTarget: EventTarget | null,
+  ) {
+    if (relatedTarget === null) {
+      // Workaround:
+      // Skip document.body/null case until this blur bugs is fixed to avoid
```

Comment:
```suggestion
      // Skip document.body/null case until this blur bug is fixed to avoid
```

Nit on extra 's'.

Reply:
Thanks, fixed in 586a6f0734847fc76e0871619aba6a615f8ae17d